site/docs/releases.md

```diff
@@ -72,10 +72,90 @@ To add a dependency on Iceberg in Maven, add the following to your `pom.xml`:
+  - Deprecate legacy ways for loading position deletes [\#11242](https://github.com/apache/iceberg/pull/11242))
+  - Parallelize manifest writing for many new files [\#11086](https://github.com/apache/iceberg/pull/11086))
+  - Support appending files with different specs [\#9860](https://github.com/apache/iceberg/pull/9860))
+* Flink
+  - Introduces the new IcebergSink based on the new V2 Flink Sink Abstraction [\#10179](https://github.com/apache/iceberg/pull/10179))
+  - Update Flink to use planned Avro reads [\#11386](https://github.com/apache/iceberg/pull/11386))
+  - Infer source parallelism for FLIP-27 source in batch execution mode [\#10832](https://github.com/apache/iceberg/pull/10832))
+  - Make FLIP-27 default in SQL and mark the old FlinkSource as deprecated [\#11345](https://github.com/apache/iceberg/pull/11345))
+  - Support limit pushdown in FLIP-27 source [\#10748](https://github.com/apache/iceberg/pull/10748))
+* GCS
+  - Refresh vended credentials [\#11282](https://github.com/apache/iceberg/pull/11282))
+* Hive
+  - Add View support for HIVE catalog [\#9852](https://github.com/apache/iceberg/pull/9852))
+* OpenAPI
+  - Add RemovePartitionSpecsUpdate REST update type [\#10846](https://github.com/apache/iceberg/pull/10846))
+  - Add endpoint to retrieve valid credentials for a given table [\#11281](https://github.com/apache/iceberg/pull/11281))
+  - Standardize credentials in loadTable/loadView responses [\#10722](https://github.com/apache/iceberg/pull/10722))
+  - Add Scan Planning Endpoints to open api spec [\#9695](https://github.com/apache/iceberg/pull/9695))
+  - Add REST Compatibility Kit [\#10908](https://github.com/apache/iceberg/pull/10908))
+* Spark 
+  - Parallelize reading files in migrate procedures [\#11043](https://github.com/apache/iceberg/pull/11043)) 
+  - Action to compute table stats [\#11106](https://github.com/apache/iceberg/pull/11106)) 
+  - Action to remove dangling deletes [\#11377](https://github.com/apache/iceberg/pull/11377))
+  - Add utility to load table state reliably [\#11115](https://github.com/apache/iceberg/pull/11115))
+  - Don't change table distribution when only altering local order [\#10774](https://github.com/apache/iceberg/pull/10774)) 
+  - Update Spark to use planned Avro reads [\#11299](https://github.com/apache/iceberg/pull/11299))
+  - Spark Action to Analyze table [\#10288](https://github.com/apache/iceberg/pull/10288))
+  - Support Column Stats [\#10659](https://github.com/apache/iceberg/pull/10659))
+  - Add RewriteTablePath action interface [\#10920](https://github.com/apache/iceberg/pull/10920))
+* Spec
+  - Add v3 types and type promotion [\#10955](https://github.com/apache/iceberg/pull/10955))
+  - Adds Row Lineage [\#11130](https://github.com/apache/iceberg/pull/11130))
+  - Deprecate the file system table scheme. [\#10833](https://github.com/apache/iceberg/pull/10833))
+
+
 ### 1.6.0 release
 Apache Iceberg 1.6.0 was released on July 23, 2024.
 
```

Comment:
1.6.1 is missing as well, but that's for another PR

Reply:
Yes, I wast trying to not put too much in this one